ops/ansible/roles/bfd-db-migrator/templates/migrator-monitor.sh.j2

```diff
@@ -65,18 +65,46 @@ function get_migrator_state {
 #######################################
 function migrator_is_running {
   local _state="$1"
-  local _code="$(jq -r .code <<< "$_state")"
-  local _status="$(jq -r .status <<< "$_state")"
+
+  local _code
+  local _status
+  _code="$(jq -r .code <<< "$_state")"
+  _status="$(jq -r .status <<< "$_state")"
   [ "$_code" = '(null)' ] && [ "$_status" = "0/0" ]
 }
 
+#######################################
+# Sets the migration schema version based on the monitor log
+# Globals:
+#   None
+# Arguments:
+#   Current state of the migrator
+# Outputs:
+#  json response to be used to update the schema version
+#  that was reported out in the monitor log and report that schema version
+#  to be used to send to SQS in order to update the migrator on whether to deploy or not
+#######################################
+function set_migrated_state {
+  local _state="$1"
+  local schema_version
+
+  if grep -q "No migration necessary." "{{ db_migrator_dir }}/migrator-log.json"; then
+    schema_version="$(grep "Current version of schema" "{{ db_migrator_dir }}/migrator-log.json" | jq -r .message | cut -d: -f2 | head -n1 | xargs)"
+  else
+    schema_version="$(grep "now at version" "{{ db_migrator_dir }}/migrator-log.json" | jq -r .message | grep -o 'v[0-9]\+' | sed 's/[^0-9]//g' | head -n1 | xargs)"
+  fi
+  jq --arg arg "$schema_version" -c '. += {"schema_version": $arg}' <<< "$_state"
+}
+
 while true; do
   currentState="$(get_migrator_state)"
-  send_message "$currentState"
   if migrator_is_running "$currentState"; then
+    send_message "$currentState"
     sleep "$MONITOR_HEARTBEAT_INTERVAL_SECONDS"
     continue
   else
+    migratedState="$(set_migrated_state "$currentState")"
+    send_message "$migratedState"
```

Comment:
do you mind updating the camel case vars to snake case? I know you didn't come up with some of them!

Reply:
done